docs/kyma-release-notes/2023-10-02-release-notes-2.19.md

```diff
@@ -0,0 +1,35 @@
+
+## Application Connectivity
+
+Additional logging options have been added to Central Application Gateway.
+https://github.com/kyma-project/kyma/#debugging
+
+## Observability
+
+Early warning: The Monitoring component will be removed with Kyma 2.20, as announced in this [blog post](https://blogs.sap.com/2023/09/07/removal-of-prometheus-grafana-based-monitoring-in-sap-btp-kyma-runtime/).
+
+Update of the software stack:
+ - [kube-state-metrics 2.10.0](https://github.com/kyma-project/kyma/pull/18135)
+ - [oauth2-proxy 7.5.1](https://github.com/kyma-project/kyma/pull/18222)
+
+## Telemetry
+
+The Telemetry component has been [replaced by the module](https://github.com/kyma-project/kyma/issues/16301) and won't be part of the Kyma release notes anymore. Instead, learn about upcoming releases of the Telemetry module on https://github.com/kyma-project/telemetry-manager/releases.
```

Comment:
```suggestion
The Telemetry component has been [replaced ](https://github.com/kyma-project/kyma/issues/16301) by the [Telemetry module](https://github.com/kyma-project/telemetry-manager) and won't be part of Kyma releases anymore. Instead, learn about upcoming releases of the Telemetry module on the dedicated [release page](ehttps://github.com/kyma-project/telemetry-manager/releases).
```

Reply:
@IwonaLanger I prefer it more like that, do you agree?

Reply:
Looks great to me 👍